src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,32 @@
 import React from 'react';
+import './CommentList.scss';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  const commentListContent = (
+    comments.map(comment => (
+      <CommentInfo
+        comment={comment}
+        key={comment.id}
+      />
+    )));
```

Comment:
Why do you need it if there will be no comments? Also, every time your component re-renders this constant updates.